pkg/apis/openebs/v1alpha1/jivavolume_types.go

```diff
@@ -24,49 +24,60 @@ import (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 type ISCSISpec struct {
-	TargetIP   string `json:"targetIP"`
-	TargetPort int32  `json:"targetPort"`
-	Iqn        string `json:"iqn"`
+	TargetIP   string `json:"targetIP,omitempty"`
+	TargetPort int32  `json:"targetPort,omitempty"`
+	Iqn        string `json:"iqn,omitempty"`
 }
 
 type MountInfo struct {
 	// StagingPath is the path provided by K8s during NodeStageVolume
 	// rpc call, where volume is mounted globally.
-	StagingPath string `json:"stagingPath"`
+	StagingPath string `json:"stagingPath,omitempty"`
 	// TargetPath is the path provided by K8s during NodePublishVolume
 	// rpc call where bind mount happens.
-	TargetPath string `json:"targetPath"`
-	FSType     string `json:"fsType"`
-	DevicePath string `json:"devicePath"`
+	TargetPath string `json:"targetPath,omitempty"`
+	FSType     string `json:"fsType,omitempty"`
+	DevicePath string `json:"devicePath,omitempty"`
 }
 
 // JivaVolumeSpec defines the desired state of JivaVolume
 // +k8s:openapi-gen=true
 type JivaVolumeSpec struct {
-	PV        string    `json:"pv"`
-	Capacity  string    `json:"capacity"`
-	ISCSISpec ISCSISpec `json:"iscsiSpec"`
-	MountInfo MountInfo `json:"mountInfo"`
+	PV       string `json:"pv"`
+	Capacity string `json:"capacity"`
+	// AccessType can be specified as Block or Mount type
+	AccessType string `json:"accessType"`
```

Comment:
Can we add `omitempty` tag  here as well as part of good practice ?

Reply:
we will add this in the next PR in which use of accessType is there.